hw08.scm

```diff
@@ -1,8 +1,7 @@
 (define (reverse lst)
   (if (null? lst)
```

Comment:
Remove the changes to these files from this PR

Reply:
Done. Though, I've just realized that this repo should probably be made private / these files removed from the history, since these are the solutions to the 61A HW.

Reply:
I can probably remove the files from the history, but it'll likely mess up your local copy of the repo.